Server/Resources.xcassets/springboard-alerts/springboard-alerts-fr.dataset/alerts.json

```diff
@@ -234,48 +234,6 @@
     ],
     "shouldAccept": true
   },
```

Comment:
`Sign In with Apple ID` missed here. Please double check

Reply:
This file does not include this alert. Thanks for the point